editor/src/core/shared/math-utils.ts

```diff
@@ -741,6 +741,22 @@ export function lineIntersection<C extends CoordinateMarker>(
   } as Point<C>
 }
 
+export function sideOfLine<C extends CoordinateMarker>(
+  lineA: Point<C>,
+  lineB: Point<C>,
+  p: Point<C>,
+): 'left' | 'right' | 'on-line' {
+  const d = (p.x - lineA.x) * (lineB.y - lineA.y) - (p.y - lineA.y) * (lineB.x - lineA.x)
```

Comment:
Would it be possible to have a little comment here explaining what's going on here please?

Reply:
Added comments in 86186a0b08b4e3fec1e02751247e6e86f33d819d